### PR TITLE
show user profile and list all articles of user

### PR DESCRIPTION
**Pull request by LeeRenJie**
depends on #23

### CODE REVIEW

File: alpha-blog/app/controllers/users_controller.rb [hunk 1]
```diff
@@ -1,6 +1,12 @@
 class UsersController < ApplicationController
+
+  def show
+    @user = User.find(params[:id])
+  end
+
   def new
     @user = User.new
+    @articles = @user.articles
```
Comment (tlkiong) [inline alpha-blog/app/controllers/users_controller.rb:9]: why can't put this in the view itself?

File: alpha-blog/app/views/articles/_article.html.erb [hunk 2]
```diff
@@ -0,0 +1,23 @@
+<div class="container">
+  <% obj.each do |article| %>
```
Comment (tlkiong) [inline alpha-blog/app/views/articles/_article.html.erb:2]: What is `obj` ?
Naming is super important as it tells us (the readers) what it represents

**Review by tlkiong — CHANGES_REQUESTED** [review]

**Review by tlkiong — COMMENTED** [review]
Can you run your code and make sure its working before ask me to review?

**Review by tlkiong — CHANGES_REQUESTED** [review]
Can you run your code after your change to ensure it can run properly before ask me to review again?

**Comment by LeeRenJie** [comment]
> Can you run your code after your change to ensure it can run properly before ask me to review again?

Works previously didn't realize the extra bracket. Apologies😅

File: alpha-blog/app/views/articles/index.html.erb [hunk 3]
```diff
@@ -1,26 +1,4 @@
...
-          </div>
-          <div class="card-footer text-muted">
-            <small>Created <%= time_ago_in_words(article.created_at) %> ago, edited <%= time_ago_in_words(article.updated_at) %> ago</small>
-          </div>
-        </div>
-      </div>
-    </div>
-  <% end %>
-</div>
+<%= render 'article', articles: @articles %>
```
Comment (tlkiong) [inline alpha-blog/app/views/articles/index.html.erb:3]: Are you 10000% sure this one works without error?

**Review by tlkiong — CHANGES_REQUESTED** [review]

File: alpha-blog/app/views/articles/index.html.erb (hunk 3 above)
Comment (LeeRenJie) [reply alpha-blog/app/views/articles/index.html.erb:3]: Yea it shows all the articles at `/articles`

Comment (LeeRenJie) [reply alpha-blog/app/views/articles/index.html.erb:3]: image

Comment (tlkiong) [reply alpha-blog/app/views/articles/index.html.erb:3]: where is `@articles` defined?

**Review by tlkiong — APPROVED** [review]

**Pull request merged**